src/common/Touchable.js

```diff
@@ -30,8 +33,35 @@ type Props = {
 export default class Touchable extends PureComponent<Props> {
   props: Props;
 
+  defaultProps: {
+    handlePressWithDelay: true,
```

Comment:
What's the motivation for adding this option?

It looks like you don't actually have any callers turn it off.